Apps/Database/src/Migrations/20230411193034_CreateDependentAuditTables.cs

```diff
@@ -0,0 +1,109 @@
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+#pragma warning disable CS1591
+// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+#pragma warning disable CA1814 // Prefer jagged arrays over multidimensional
+
+namespace HealthGateway.Database.Migrations
+{
+    /// <inheritdoc />
+    public partial class CreateDependentAuditTables : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "DependentAuditOperationCode",
+                schema: "gateway",
+                columns: table => new
+                {
+                    Code = table.Column<string>(type: "character varying(10)", maxLength: 10, nullable: false),
+                    Description = table.Column<string>(type: "character varying(50)", maxLength: 50, nullable: false),
+                    CreatedBy = table.Column<string>(type: "character varying(60)", maxLength: 60, nullable: false),
+                    CreatedDateTime = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
+                    UpdatedBy = table.Column<string>(type: "character varying(60)", maxLength: 60, nullable: false),
+                    UpdatedDateTime = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
+                    xmin = table.Column<uint>(type: "xid", rowVersion: true, nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_DependentAuditOperationCode", x => x.Code);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "DependentAudit",
+                schema: "gateway",
+                columns: table => new
+                {
+                    DependentAuditId = table.Column<Guid>(type: "uuid", nullable: false),
+                    DelegateHdId = table.Column<string>(type: "character varying(52)", maxLength: 52, nullable: false),
+                    AgentUsername = table.Column<string>(type: "character varying(255)", maxLength: 255, nullable: false),
+                    ProtectedReason = table.Column<string>(type: "character varying(500)", maxLength: 500, nullable: false),
```

Comment:
Nit: A more `Reason` would be a better futureproof property name, as currently we're only capturing protection but that doesn't mean we won't have other audited actions in the future.